onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -4,7 +4,7 @@
 
 import onnxscript.ir as ir
 from onnxscript.ir.passes.common import shape_inference
-from onnxscript.optimizer import optimize, remove_unused_nodes
+from onnxscript.optimizer import optimize
 from onnxscript.rewriter import rewrite
 from onnxscript.rewriter.ort_fusions import (
```

Comment:
Would it be possible to import modules only? This is only to help prevent import cycles down the line

Reply:
This file is supposed to depend on most of the remaining files (but other files should not depend on this). So, I am not sure replacing 10 odd lines by 20 is worth it (because we'd like to expose imported fuse_X methods in __init__ also ... at the least, it seems worth doing it) where "from X import y" becomes "import X"; "y = X.y" ...